File: VB/CustomLocalizer/MyEditorsLocalizer.vb
Imports DevExpress.Web.Localization
Imports DevExpress.Utils.Localization.Internal

Namespace CustomLocalizer

    Public Class MyEditorsLocalizer
        Inherits ASPxEditorsLocalizer

        Public Shared Sub Activate()
            Dim localizer As New MyEditorsLocalizer()
            Dim provider As DefaultActiveLocalizerProvider(Of ASPxEditorsStringId) = _
                New DefaultActiveLocalizerProvider(Of ASPxEditorsStringId)(localizer)
            MyEditorsLocalizer.SetActiveLocalizerProvider(provider)
        End Sub

        Public Overrides Function GetLocalizedString(ByVal id As ASPxEditorsStringId) As String
            Dim result As String = MyBase.GetLocalizedString(id)
            Return String.Format("editors:{0}", result)
        End Function

    End Class

End Namespace

File: CS/CustomLocalizer/MyEditorsLocalizer.cs
using DevExpress.Utils.Localization.Internal;
using DevExpress.Web.Localization;

namespace CustomLocalizer {
    public class MyEditorsLocalizer : ASPxEditorsLocalizer {

        public static void Activate() {
            MyEditorsLocalizer localizer = new MyEditorsLocalizer();
            DefaultActiveLocalizerProvider<ASPxEditorsStringId> provider =
                new DefaultActiveLocalizerProvider<ASPxEditorsStringId>(localizer);
            MyEditorsLocalizer.SetActiveLocalizerProvider(provider);
        }

        public override string GetLocalizedString(ASPxEditorsStringId id) {
            string result = base.GetLocalizedString(id);
            return string.Format("editors:{0}", result);
        }

    }
}
